bosch_thermostat_client/gateway/base.py

```diff
@@ -63,6 +66,7 @@ def __init__(self, host):
         self._initialized = None
         self.initialization_msg = None
         self._bus_type = None
+        self_errors = None
```

Comment:
ITYM?

```suggestion
        self._errors = None
```

Reply:
Oops, thanks.
Didn't fail because it gets overwritten in the inherited classes.